### PR TITLE
add get current branch snippet

### PR DESCRIPTION
**Pull request by nagauta**

### CODE REVIEW

File: data/snippets.ts [hunk 1]
```diff
@@ -805,6 +805,13 @@ const misc = [
     keyword: "cal",
     type: "template",
   },
+  {
+    id: nanoid(),
+    name: "Get Name of Current Branch",
+    text: "git branch --contains | cut -d \" \" -f 2",
+    keyword: "git-current-branch",
+    type: "template",
+  },
```
[the remark this answers is not in the document]
Comment (nagauta) [reply data/snippets.ts:814]: Thanx!, yeah, "Coding" Section is better, I'm fixin' this!

**Review by peduarte — APPROVED** [review]

**Comment by peduarte** [comment]
Very nice, thank you! Can you resolve conflicts and we're good to go.

File: data/snippets.ts [hunk 2]
```diff
@@ -530,6 +530,12 @@ const coding = [
     keyword: "sql-select",
     type: "template",
   },
+  {
+    id: nanoid(),
+    name: "Get Name of Current Branch",
+    text: "git branch --contains | cut -d \" \" -f 2",
+    keyword: "git-current-branch",
```
Comment (peduarte) [inline data/snippets.ts:537]: missing `type: "template",`

```suggestion
    keyword: "git-current-branch",
    type: "template",    
```

**Review by peduarte — CHANGES_REQUESTED** [review]
missing `type`

**Review by peduarte — APPROVED** [review]

File: data/snippets.ts [hunk 3]
```diff
@@ -530,6 +530,12 @@ const coding = [
     keyword: "sql-select",
     type: "template",
   },
+  {
+    id: nanoid(),
+    name: "Get Name of Current Branch",
+    text: "git branch --contains | cut -d \" \" -f 2",
```
Comment (aashutoshrathi) [inline data/snippets.ts:536]: Shouldn't we use `git branch --show-current`?

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]

File: data/snippets.ts (hunk 3 above)
Comment (peduarte) [reply data/snippets.ts:536]: If that's a better way, feel free to create a PR. We can change it.

Comment (nagauta) [reply data/snippets.ts:536]: @aashutoshrathi 
Thanks. It’s better!

**Pull request merged**